build.proj

```diff
@@ -185,19 +185,24 @@
              Properties="TargetFramework=$(New3TargetFramework);Configuration=$(Configuration);VSTestLogger=trx"
              RunEachTargetSeparately="true" StopOnFirstFailure="true" />
 
-	</Target>
-	
+    </Target>
+    
   <!-- Localization -->
   <Target Name="BuildLocalized" DependsOnTargets="SetLanguages" Condition="Exists('.tools\loc\loc.proj')">
     <MSBuild Projects=".tools\loc\loc.proj" Targets="Build"
              Properties="Lang=%(Languages.Identity)" />
-			 	
-	<MSBuild Condition="'$(PB_SignType)' == 'real' or '$(PB_SignType)' == 'test'"
+
+    <!-- String name the localized assemblies-->
```

Comment:
Typo :) should be Strongname, not String name

Reply:
oops. Will fix it.